Doc/library/sys.rst

```diff
@@ -1300,6 +1300,9 @@ always available.
    Note that as an exception is propagated down the chain of callers, an
    ``'exception'`` event is generated at each level.
 
+   For more fine-grained usage, it's possible to set a trace function for
+   each frame individually rather than all code, by setting 
```

Comment:
It looks like this line has a trailing space (which is breaking CI). Do you mind removing it?

```suggestion
   each frame individually rather than all code, by setting
```

Reply:
Fixed.